_research/ai-models/cardiac-pathology/2026-pattern-identification-cvds.md

```
@@ -0,0 +1,203 @@
+![Figure text](https://github.com/modelflows/modelflowsapp/blob/master/assets/img/2025_01Jan_30_Bell_DiagnosisOverview.jpg?raw=true)
+
+
+The results obtained show the efficacy of the HODMD algorithm and the superiority of the proposed system, even outperforming pretrained Convolutional Neural Networks (CNNs), the method of choice in the literature. 
+
+Further details about the work explained in this section could be found in the following reference:
+
+[*Bell-Navas, A., Groun, N., Villalba-Orero, M., Lara-Pezzi, E., Garicano-Mena, J., & Le Clainche, S., 2025. “Automatic Cardiac Pathology Recognition in Echocardiography Images using Higher Order Dynamic Mode Decomposition and a Vision Transformer for Small Datasets.” Exp. Syst. Appl., 264, 125849.*](https://doi.org/10.1016/j.eswa.2024.125849)
+
+<!-- The following video summarizes the developed system and shows the obtained results in the diagnosis task: -->
+
+Download the code for **Diagnosis** in Python version [*here*](https://github.com/modelflows/ModelFLOWs-cardiac/raw/refs/heads/main/legacy-medical-data-tutorials/Diagnosis_scripts.zip).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)